### DISCUSSION TITLE
PLaytime precount at every start of the arrangement window loop

### DISCUSSION THREAD

**Discussion by dantec3p** [helgoboss/helgobox#1161] [Q&A]
Hi Benjamin, congrats for your work, and hi everyone.
I do live music with Playtime 2 but also with looped regions and the smooth seeking in the arrangement window.
When I'm going from a point to another (I mean when I "jump" in the timeline), Playtime 2, instead of keeping looping the clips as I wish it would, triggers the precount and launches the clip one measure later (depending on my setting of the start timing I guess).

So music from PLaytime come one measure after the rest of the sounds!

Is this normal? Am I the only one tu use both clips AND looped regions in the arrangement window for live music?

Hope there is some way to counter this... ?
Have a good day

**Comment by helgoboss** [comment]
Hi @dantec3p! You want https://github.com/helgoboss/helgobox/discussions/1004.

**Reply by dantec3p** [reply]
yes I guess that's it, I had not seen this 1004...
Indeed I think some clips just keeping playing and some others are triggered (with precount). Strange...

**Reply by helgoboss** [reply]
It's just not supported yet. REAPER has many features and it's kind of impossible to support all of them in combination with Playtime. But this one is one of them for which I explicitly plan to add support.

**Reply by helgoboss** [reply]
This (and smooth seek) is supported in 2.16.4, but for now only for non-PDC tracks.

**Comment by helgoboss** [comment]
@dantec3p Smooth seek support will be implemented in 2.16.4. In addition, I will keep Playtime's clips Playing at loop boundaries.
